### ISSUE TITLE
[SUGGESTION] Sleeping bag mod suggestion

### ISSUE THREAD

**Issue by ThePoleThatFishes** [Gliese-832-c/ad-astra-per-nucleon#19]
**Is your suggestion related to a problem? Please describe.**
Sleeping away from home without resetting spawn point :(

**Describe the solution you'd like**
Add this [mod](https://www.curseforge.com/minecraft/mc-mods/comforts)?

**Comment by Gliese-832-c** [comment]
Any suggestions on how to maybe make it more expensive to balance it?

**Comment by ThePoleThatFishes** [comment]
You'll be using it out in nature, so some protective fiber/treated leather/whatever from TG should be used in its construction.
I've noticed it has some configs for negative effects too, so perhaps give slight debuffs for not sleeping well?

**Comment by joendter** [comment]
Like nausea for all day?

**Comment by ThePoleThatFishes** [comment]
No, that would be way too annoying. Probably something like weakness/slowness for 30s-1 min after waking up.

**Comment by ThePoleThatFishes** [comment]
I have downloaded the mod in question and took a look at the config;
It has config options such as sleeping bag having a chance to break upon usage (0% by default), negative effects upon using sleeping bag & a "sleepy factor" which when enabled, allows you to sleep every x amount of time.
@Gliese-832-c what do you think?

**Comment by Gliese-832-c** [comment]
I agree, I'll do something like that.
